ui/js/controllers/perf/compare.js

```diff
@@ -1,10 +1,34 @@
 "use strict";
 
+perf.value('compareDefaultTimeRange', 86400 * 2);
+
+perf.factory('compareFactory',[
```

Comment:
Let's just call this verifyRevision and just return a function. 'compareFactory' is a very vague name.